manticore/platforms/linux.py

```diff
@@ -1047,12 +1053,12 @@ def sys_write(self, fd, buf, count):
 
             if not self._is_open(fd):
                 logger.error("WRITE: Not valid file descriptor. Returning EBADFD %d", fd)
-                return errno.EBADF
+                return -errno.EBADF
```

Comment:
i just checked the man page and write(2) returns -1 on error, setting `errno` to the error code; it doesn't return the err code.